packages/orbit-components/src/Textarea/index.tsx

```diff
@@ -78,9 +78,8 @@ const StyledTextArea = styled.textarea<StyledTextAreaProps>`
   padding: ${getPadding};
   box-shadow: inset 0 0 0
     ${({ theme, error }) =>
-      `${theme.orbit.borderWidthInput} ${
-        error ? theme.orbit.borderColorInputError : theme.orbit.borderColorInput
-      }`};
+    `${theme.orbit.borderWidthInput} ${error ? theme.orbit.borderColorInputError : theme.orbit.borderColorInput
```

Comment:
@RobinCsl if you already touched that, could you please refactor the component to a single destructured? 🙏🏻 :)

Reply:
auto-formatting touched it, does that count? :D

Reply:
done

Reply:
interesting 😮 it should not happen

Reply:
I don't think I have it installed globally, it might have been my Neovim/LSP setup :)